iphone/Classes/TIUIiOSAlertDialogStyleProxy.h

```diff
@@ -0,0 +1,22 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2012 by Appcelerator, Inc. All Rights Reserved.
```

Comment:
Copyright of all new files to 2016. Check all other new files you created as well.